vtc.am

```diff
@@ -8,7 +8,7 @@
 # The current assumption is that all the tests for the Makefile are VTCs
 # and may be revisited if needed.
 
-VTC_LOG_COMPILER = $(top_builddir)/bin/varnishtest/varnishtest -v -i
+VTC_LOG_COMPILER = $(top_builddir)/bin/varnishtest/varnishtest -v -i -D PYTHON=$(PYTHON)
```

Comment:
Should `varnishtest -i` discover and define `${PYTHON}`?

Reply:
if this goes in, we might as well. Python is stable, widespread and useful enough that I feel this is warranted.
But I'm a bit worried of reimplementing the discovery in a way that doesn't match `autotools`'

Reply:
You are correct, and I definitely don't want to diverge. We can always ask nicely:

```
grep ^PYTHON= config.log
```